public/static/locales/en/login.json

```diff
@@ -2,28 +2,28 @@
   "title": "Sign in",
   "emailLabel": "Email",
   "signInKeyLabel": "Form sign-in key",
-  "signInKeyDescription": "Enter the form sign in key emailed to you when you were provided access to the form.",
+  "signInKeyDescription": "Enter the form sign-in key emailed to you when you were provided access to the form.",
   "signInKeyForgot": "Forgot your form sign-in key?",
   "signInKeyReset": "Request a reset (opens in new tab)",
-  "loginInvalidCredentialsErrorTitle": "Account details are incorrect",
-  "loginInvalidCredentialsErrorMessage": "Form sign-in key and email do not match our records.",
+  "loginInvalidCredentialsErrorTitle": "Account details are incorrect.",
+  "loginInvalidCredentialsErrorMessage": "Double-check your email and form sign-in key are correct",
   "loginEmailErrorMessage": "Double-check your email before trying again.",
-  "loginSignInKeyErrorMessage": "Double-check your sign-in key before trying again.",
+  "loginSignInKeyErrorMessage": "Double-check the form sign-in key before trying again.",
   "loginInternalErrorTitle": "There was a problem with the web page",
   "loginInternalErrorMessage": "Please try again.",
-  "remainingNumberOfLoginAttemptsErrorMessagePart1": "Account will be locked after",
-  "remainingNumberOfLoginAttemptsErrorMessagePart2": "more tries.",
-  "lockoutErrorTitle": "Account locked",
-  "lockoutDurationErrorMessagePart1": "Wait",
-  "lockoutDurationErrorMessagePart2": "seconds before trying to sign in again.",
+  "remainingNumberOfLoginAttemptsErrorMessagePart1": "You have tried to sign in",
+  "remainingNumberOfLoginAttemptsErrorMessagePart2": "times. This account will be locked after 10 tries.",
+  "lockoutErrorTitle": "Account locked.",
+  "lockoutDurationErrorMessagePart1": "Sign-in will be blocked for",
+  "lockoutDurationErrorMessagePart2": "seconds.",
   "temporaryToken": {
     "title": "Check your email",
-    "instructions": "We've sent you a one-time-use, 6-digit verification code by email. It may take a a few minutes to arrive.",
+    "instructions": "We've sent you a one-time-use, 6-digit verificartion code by email. It may take a a few minutes to arrive.",
     "temporaryTokenLabel": "Enter verification code",
     "submitButton": "Submit",
-    "helpText": "Didn't get the verification code?",
+    "helpText": "Didn't get the verification code ? ",
```

Comment:
This is an odd change, I don't think it's convention to have a space around a question mark at the end of a sentence.